src/test/scala/org/constellation/util/wallet/CreateNewTransactionTest.scala

```diff
@@ -0,0 +1,139 @@
+package org.constellation.util.wallet
+import better.files.File
+import cats.effect.IO
+import org.constellation.Fixtures
+import org.constellation.keytool.KeyStoreUtils
+import org.constellation.primitives.Transaction
+import org.scalatest._
+import constellation._
+
+class CreateNewTransactionTest extends AsyncFlatSpecLike with Matchers with BeforeAndAfterAll with BeforeAndAfterEach {
+  val walletClient = CreateNewTransaction
+  val accountPath = "src/test/resources/valid-tx.txt"
```

Comment:
instead of hardcoding directories, following snippet should work: `getClass.getResource("valid-tx").getPath`